lib/pure/browsers.nim

```diff
@@ -78,16 +73,31 @@ proc openDefaultBrowser*(url: string) =
   ##   block: openDefaultBrowser("https://nim-lang.org")
   ##   ```
   doAssert url.len > 0, "URL must not be empty string"
-  openDefaultBrowserImpl(url)
+  openDefaultBrowserRaw(url)
 
-proc openDefaultBrowser*() {.since: (1, 1).} =
-  ## Opens the user's default browser without any `url` (blank page). This does not block.
-  ## Implements IETF RFC-6694 Section 3, "about:blank" must be reserved for a blank page.
+import std/private/since
```

Comment:
Please move it back where it was in the line 15

Reply:
I thought this would be more notable and easier when it's time to remove that proc,

as `since` is only used by this proc.

...Well, I've put it back, adding a comment about it